openmetadata-ui/src/main/resources/ui/src/pages/tags/index.tsx

```diff
@@ -632,6 +807,20 @@ const TagsPage = () => {
                     onSave={(data) => createPrimaryTag(data as TagCategory)}
                   />
                 )}
+                {deleteTags.state && (
+                  <ConfirmationModal
+                    bodyText={`Are you sure you want to delete the tag ${
+                      deleteTags.data?.isCategory ? 'category' : ''
+                    } "${deleteTags.data?.name}"?`}
+                    cancelText="Cancel"
+                    confirmText="Confirm"
+                    header="Delete Team"
```

Comment:
```suggestion
                    header="Delete Tag"
```

Reply:
@Sachin-chaurasiya nice catch on this, thanks for highlighting.